sdk/containerregistry/Azure.Containers.ContainerRegistry/src/Models/AcrManifests.cs

```diff
@@ -10,11 +10,11 @@ namespace Azure.Containers.ContainerRegistry
     internal partial class AcrManifests
     {
         /// <summary> List of manifests. </summary>
-        public IReadOnlyList<RegistryArtifactProperties> RegistryArtifacts
+        public IReadOnlyList<ArtifactManifestProperties> RegistryArtifacts
         {
             get
             {
-                List<RegistryArtifactProperties> artifacts = new List<RegistryArtifactProperties>();
+                List<ArtifactManifestProperties> artifacts = new List<ArtifactManifestProperties>();
```

Comment:
Perf could be better if you pass `this.Manifests.Count` (or `.Length` depending on what it is) to the constructor to initialize capacity. Depending on how many manifests are contained herein, this could save quite a bit of memory and CPU time.

Reply:
Thanks - will do!